book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    @rentals << Rental.new(date, person)
```

Comment:
- [ ] @masangana great work you should also add the book itself as a parameter for creating the new rental.
`hint` you can use `self` to represent the book.